lib/matplotlib/backend_bases.py

```diff
@@ -2644,23 +2644,19 @@ class FigureManagerBase(object):
     num : int or str
         The figure number
 
+    key_press_handler_id : int
```

Comment:
Shouldn't this whole bit of docstring be under `__init__` below? I don't think either place is right for `key_press_handler_id` anyway, since neither method takes that as a parameter.

Reply:
It's documented as a class attribute, not as a constructor parameter.